src/graphql-server/resolvers.js

```diff
@@ -60,21 +60,15 @@ export const resolvers = {
 
     hposStatus: HposInterface.os.status,
 
-    hostingReport: () => {
+    hostingReport: async () => {
+      const hostedHapps = await HposInterface.os.hostedHapps()
+
+      const localSourceChains = hostedHapps.reduce((total, happ) => total + happ.number_instances, 0)
+
       return {
-        localSourceChains: 18,
+        localSourceChains,
         zomeCalls: 588,
```

Comment:
Let's also add this mock data to the hpos mocks.

Reply:
This shouldn't go in the mocks yet because we haven't implemented the actual API calls yet. We should just wait til we have the real calls before creating "real" mocks.